app/src/main/res/layout/fragment_settings.xml

```diff
@@ -7,27 +7,42 @@
     android:orientation="vertical"
     android:theme="@style/SettingsTheme">
 
+    <androidx.constraintlayout.widget.Guideline
```

Comment:
This worth extracting as an `<include>` since it's getting used in multiple spots?

Reply:
Good call, I updated to use an include.